<launch>
	
        <group ns="pioneer1">
    		<param name="tf_prefix" value="pioneer1" />
		<node name="RelocNode_tcp" pkg="RelocSensorDriver" type="RelocNode_tcp1.py" args="192.168.1.1 5000" />
	</group>

	<!-- Copy as necessary -->
	
	<!-- node name="rviz" pkg="rviz" type="rviz" args="-d $(find ariaClientDriver)/ariaClientDriver5.vcg" /-->
	
</launch>
